### PR TITLE
TER-89 check too many columns

### PR DESCRIPTION
**Pull request by martinjunger**
Jira: TER-89
Connection: https://github.com/keboola/connection/pull/4199

Osetreni chyb:
- 3919   Table has too many columns. ([doc](https://docs.teradata.com/r/bBJcqMYyoxECDlJRAz9Dgw/41k1Cl3VSGevNbfgvlZE3Q))
- 3933   The available data bytes in the table's perm row has exceeded the %VSTR byte limit. ([doc](https://docs.teradata.com/r/bBJcqMYyoxECDlJRAz9Dgw/41k1Cl3VSGevNbfgvlZE3Q))

### CODE REVIEW

File: src/Handler/Exception/ExceptionResolver.php [hunk 1]
```diff
@@ -25,6 +25,12 @@ public static function resolveException(\Throwable $e): \Throwable
                 if (strpos($e->getMessage(), 'No more room in database') !== false) {
                     return NoSpaceException::createForFullDB($e);
                 }
+                if (strpos($e->getMessage(), "available data bytes in the table's perm row has exceeded")) {
```
Comment (jirkasemmler) [inline src/Handler/Exception/ExceptionResolver.php:28]: kdyz budeme predpokladat, ze driver si ma take ocheckovat pocet sloupcu, tak za me to nema delat retrospektivne, ale checkem pred pustenim query. a odpoved by imho mela byt odpovidajici s `ColumnCollection`

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
ten resolver je asi cajk. kdyz to chcipne, tak at to dopadne hezkou exceptiou. Ale melo by se to checkovat pred volanim

File: src/Handler/Exception/ExceptionResolver.php [hunk 2]
```diff
@@ -25,6 +25,12 @@ public static function resolveException(\Throwable $e): \Throwable
...
+                    return RowSizeTooLargeException::createForTablePermRowHasExceededLimit($e);
+                }
+                if (strpos($e->getMessage(), 'Table has too many columns')) {
```
Comment (jirkasemmler) [inline src/Handler/Exception/ExceptionResolver.php:31]: dejme tomu, ze to tady v tom resolveru nechame. Tak prosim at ta message je aspon shodna s message v ColumnsCollection

Comment (martinjunger) [reply src/Handler/Exception/ExceptionResolver.php:31]: upraveno

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**